build.sh

```diff
@@ -6,7 +6,7 @@ fi
 
 ## Build infrastructure ##
 docker-compose up -d
-sleep 10
+sleep 15
```

Comment:
How about instead, we use health checks?

Reply:
Good point, I'm searching for an alternative for this hacks